src/imageTools/angleTool.js

```diff
@@ -8,6 +9,8 @@ import drawHandles from '../manipulators/drawHandles.js';
 import { getToolState } from '../stateManagement/toolState.js';
 import lineSegDistance from '../util/lineSegDistance.js';
 import { getNewContext, draw, setShadow, drawLines } from '../util/drawing.js';
+import triggerEvent from '../util/triggerEvent.js';
```

Comment:
I see that we're importing `EVENTS` and `triggerEvent` to all of these tools so we can create a `fireCompleted`. Can we instead create a helper method that we import?

`triggerMeasurementCompletedEvent(element, toolType, data)`

Reply:
yes, good catch!

Reply:
this is now updated